src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -423,9 +425,13 @@ void FASTCALL gfx_draw_sprite_software(rct_drawpixelinfo* dpi, ImageId imageId,
  * x (cx)
  * y (dx)
  */
+
 void FASTCALL gfx_draw_sprite_palette_set_software(
-    rct_drawpixelinfo* dpi, ImageId imageId, int32_t x, int32_t y, const PaletteMap& paletteMap)
+    rct_drawpixelinfo* dpi, ImageId imageId, const ScreenCoordsXY& coords, const PaletteMap& paletteMap)
 {
+    int32_t x = coords.x;
+    int32_t y = coords.y;
+
```

Comment:
```suggestion
```

No need to do this, we can just shift them in place below

Reply:
At lines 469, 470, the values x and y are modified.

Reply:
Ah, I see.